### PR TITLE
Allow refetching instances for healthcheck

### PR DESCRIPTION
**Pull request by OlgaMaciaszek**
Fixes gh-842.

### CODE REVIEW

File: docs/src/main/asciidoc/_configprops.adoc [hunk 1]
```diff
@@ -34,6 +30,9 @@
 |spring.cloud.loadbalancer.health-check.initial-delay | 0 | Initial delay value for the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.interval | 25s | Interval for rerunning the HealthCheck scheduler.
 |spring.cloud.loadbalancer.health-check.path |  | 
+|spring.cloud.loadbalancer.health-check.refetch-instances | false | Indicates whether the instances should be refetched by the <code>HealthCheckServiceInstanceListSupplier</code>. This can be used if the instances can be updated and the underlying delegate does not provide an ongoing flux.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/_configprops.adoc:33]: Add backticks (for code) around "false". Change the last sentence to "You can use this setting if the instances can be updated and the underlying delegate does not provide an ongoing flux."

File: docs/src/main/asciidoc/_configprops.adoc [hunk 2]
```diff
@@ -34,6 +30,9 @@
...
+|spring.cloud.loadbalancer.health-check.refetch-instances-interval | 25s | Interval for refetching available service instances.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/_configprops.adoc:34]: Add backticks (for code) around "25s".

File: docs/src/main/asciidoc/_configprops.adoc [hunk 3]
```diff
@@ -34,6 +30,9 @@
...
+|spring.cloud.loadbalancer.health-check.repeat-health-check | true | Indicates whether health checks should keep repeating. It might be useful to set it to <code>false</code> if periodically refetching the instances, as every refetch will also trigger a healthcheck.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/_configprops.adoc:35]: Add backticks (for code) around "true". Change the last sentence to "If you want to periodically refetch the instances, set it to `false`, as every refetch also triggers a health check."

If you want to periodically refetch the instances, set it to `false`, as every refetch also triggers a health check

File: docs/src/main/asciidoc/spring-cloud-commons.adoc [hunk 4]
```diff
@@ -987,7 +993,7 @@ public class CustomLoadBalancerConfiguration {
 	}
 ----
 
-NOTE:: `HealthCheckServiceInstanceListSupplier` has its own caching mechanism based on Reactor Flux `replay()`, therefore, if it's being used, you may want to skip wrapping that supplier with `CachingServiceInstanceListSupplier`.
+WARNING: `HealthCheckServiceInstanceListSupplier` has its own caching mechanism based on Reactor Flux `replay()`, therefore, if it's being used, you may want to skip wrapping that supplier with `CachingServiceInstanceListSupplier`.
```
Comment (Buzzardo) [inline docs/src/main/asciidoc/spring-cloud-commons.adoc:996]: Change "`replay()`, therefore," to "`replay()`. Therefore," (two sentences).

**Review by Buzzardo — CHANGES_REQUESTED** [review]

**Review by ryanjbaxter — APPROVED** [review]

File: docs/src/main/asciidoc/_configprops.adoc (hunk 1 above)
Comment (OlgaMaciaszek) [reply docs/src/main/asciidoc/_configprops.adoc:33]: This file is automatically generated.

File: docs/src/main/asciidoc/_configprops.adoc (hunk 2 above)
Comment (OlgaMaciaszek) [reply docs/src/main/asciidoc/_configprops.adoc:34]: This file is automatically generated.

File: docs/src/main/asciidoc/_configprops.adoc (hunk 3 above)
Comment (OlgaMaciaszek) [reply docs/src/main/asciidoc/_configprops.adoc:35]: This file is automatically generated.

**Pull request merged**